src/OpenFeature/Api.cs

```diff
@@ -82,14 +82,14 @@ public FeatureProvider GetProvider()
         }
 
         /// <summary>
-        /// Gets the feature provider with given clientName
+        /// Gets the feature provider with given domain
         /// </summary>
-        /// <param name="clientName">Name of client</param>
-        /// <returns>A provider associated with the given clientName, if clientName is empty or doesn't
+        /// <param name="domain">Name of client</param>

```

Comment:
We are renaming from `clientName` to `domain`. Does it make sense to change the description accordingly?
```suggestion
        /// <param name="domain">An identifier which logically binds clients with providers</param>

```

Reply:
Totally. I missed this one.